services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/sidecar.py

```diff
@@ -48,6 +49,8 @@ def _get_environment_variables(
     *,
     allow_internet_access: bool,
     metrics_collection_allowed: bool,
+    telemetry_enabled: bool,
+    primary_group_id: GroupID,
```

Comment:
still find sad that you need to now add the PrimaryGroupID here.it's an unnecessary dependency.

Reply:
removed it